src/steamship/client/steamship.py

```diff
@@ -77,65 +73,46 @@ def embed_and_search(
             expect=QueryResults,
         )
 
-    @staticmethod
     def use(
+        self,
         package_handle: str,
         instance_handle: Optional[str] = None,
         config: Optional[Dict[str, Any]] = None,
         version: Optional[str] = None,
         fetch_if_exists: bool = True,
-        workspace_handle: Optional[str] = None,
-        **kwargs,
+        recreate: Optional[bool] = False,
     ) -> PackageInstance:
         """Creates/loads an instance of package `package_handle`.
 
-        The instance is named `instance_handle` and located in the Workspace named `instance_handle`. If no
-        `instance_handle` is provided, the default is `package_handle`.
+        The instance is named `instance_handle` and located in the workspace this client is anchored to.
+        If no `instance_handle` is provided, the default is `package_handle`.
```

Comment:
I think we should figure out how we want to generally steer people into a new workspace per package before we remove the static methods.